test/topology_custom/test_change_rpc_ip.py

```diff
@@ -0,0 +1,78 @@
+#
+# Copyright (C) 2023-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+"""
+Test clusters can restart fine after an RPC IP address change.
+"""
+
+import time
+import asyncio
+import logging
+
+import pytest
+
+from test.topology.util import wait_for_token_ring_and_group0_consistency
+
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture
+async def two_node_cluster(manager):
+    logger.info(f"Booting initial 2-node cluster")
+    servers = [(await manager.server_add()).server_id for _ in range(2)]
+    for node in servers:
+        await manager.server_start(node)
+    await wait_for_token_ring_and_group0_consistency(manager, time.time() + 30)
+    return servers
+
+
+@pytest.mark.asyncio
+async def test_change_rpc_ip(two_node_cluster, manager, random_tables):
+    """Sequentially stop two nodes, change their RPC IPs and start, check the cluster is functional."""
+
+    table = await random_tables.add_table(ncolumns=5)
+
+    # Scenario 1
+    # ----------
+    logger.info("Change RPC IP addresses consequently")
```

Comment:
nitpick: sequentially, not consequently.

Reply:
done